charts/drax/values.yaml

```diff
@@ -0,0 +1,851 @@
+  #
+  kubeIp: "10.20.20.20"
+
+  # 4G/5G Options
+  #   Description:  Depending on the product bought, you may enable or disable the accompanying features.
+  #
+  #   Value type:   bool
+  #
+  enable4G: false
+  enable5G: true
+
+  # Accelleran License
+  #   Description:  In order to use the Accelleran products, you need to have a license from Accelleran.
+  #                 Please contact Accelleran to get the license.
+  #                 The license file should be saved as a Kubernetes secret. The name of the secret should be
+  #                 supplied here under the "licenseSecretName" field.
+  #                 To create the secret you can use the following command:
+  #                 kubectl create secret generic accelleran-license --from-file=license.crt
+  #
+  # Value type:     enabled: string
+  #                 licenseSecretName: string
+  #
+  accelleranLicense:
+    enabled: "true"
+    licenseSecretName: "accelleran-license"
+
+
+bootstrap:
+  create: true
+  name: "{{ $.Release.Name }}-bootstrap"
+  redis:
+    enabled: true
+    hostname: ""
+    port: 0
+  nats:
+    enabled: true
+    hostname: ""
+    port: 0
+  kafka:
+    enabled: true
+    hostname: ""
+    port: 0
+
+
+dash-front-back-end:
+  enabled: true
+
+  config:
+    # AUTOGENERATED DONT EDIT
+    config_id: "production"
+
+    # The namespace where the L3 pod will be running
+    defaultServiceNamespace: "default"
+
+    # The namespace where the ORAN-CLUSTERCONTROLLER will be running
+    defaultOranNamespace: "default"
+
+    # AUTOGENERATED DONT EDIT
+    grafanaURL: "{{ .Values.global.kubeIp }}"
+
+    # The external nodePort used by Grafana, default is 30300
+    grafanaPort: "30300"
+
+    # AUTOGENERATED DONT EDIT
+    nodeApiURL: "{{ .Values.global.kubeIp }}"
+    nodePort: "31315"
+    kafkaUrl: "{{ .Release.Name }}-kafka"
+    kafkaPort: "9092"
+    svcOrchestratorHost: "{{ .Release.Name }}-service-orchestrator"
+    svcOrchestratorPort: "80"
+    networkStateMonitorHost: "{{ .Release.Name }}-network-state-monitor"
+    networkStateMonitorPort: "5000"
+    configApiHost: "{{ .Release.Name }}-config-api"
+    configApiPort: "80"
+    svcMonitorHost: "{{ .Release.Name }}-service-monitor"
+    svcMonitorPort: "80"
+    pcixAppPodName: "accelleran-drax-pci-010-pci-xapp-api"
+    pcixAppPort: "80"
+    ksqldbPodName: "{{ .Values.global.kubeIp }}"
+    ksqldbPort: "30533"
+    LteRadioControllerLabel: "4G-Radio-Controller"
+    draxVersionConfigmap: "{{ $.Release.Name }}-version"
+
+    # The IP that is set here must be globally available so that any browser can reach it, so that the front end can communicate to the backend. Preferable, use the publicly exposed IP that can reach the Kubernetes Advertise IP. If only a private Kubernetes Advertise IP is set here, the dRAX Dashbaord will only work when openning a browser on a computer that can reach the private Kuebrnetes Advertise IP.
+    apiUrl: "{{ .Values.global.kubeIp }}"
```

Comment:
I thought this was something which we resolved in RIC 6.5. @dimgogos do you recall the details? I have a recollection that some config fields weren't needed anymore.